doc/source/visualizing/callbacks.rst

```diff
@@ -886,3 +890,57 @@ to apply filters.
    p.frb.apply_gauss_beam(sigma=30)
    p.refresh()
    p.save()
+
+
+.. _extend-annotations:
+
+
+Extending annotations methods
+-----------------------------
+
+New ``annotate_`` methods can be added to plot object at runtime (i.e., without
+modifying yt's source code) by subclassing the base ``PlotCallback`` class.
+This is the recommended way to add custom and unique annotations to yt plots.
```

Comment:
```suggestion
New ``annotate_`` methods can be added to plot objects at runtime (i.e., without
modifying yt's source code) by subclassing the base ``PlotCallback`` class.
This is the recommended way to add custom and unique annotations to yt plots,
as it can be done through local plugins, individual scripts, or even external packages
```